pytest_mozwebqa/pytest_mozwebqa.py

```diff
@@ -137,7 +135,7 @@ def pytest_runtest_setup(item):
 
 def pytest_runtest_teardown(item):
     if hasattr(TestSetup, 'selenium') and TestSetup.selenium and 'skip_selenium' not in item.keywords:
```

Comment:
Possibly, but I don't want to change this here. Please feel free to write a patch and run some tests.